pandas/core/frame.py

```diff
@@ -1770,9 +1770,10 @@ def memory_usage(self, index=True, deep=False):
                             index=['Index']).append(result)
         return result
 
-    def transpose(self):
+    def transpose(self, *args, **kwargs):
         """Transpose index and columns"""
-        return super(DataFrame, self).transpose(1, 0)
+        nv.validate_transpose(args, kwargs, method='args')
```

Comment:
these should already *know* what to validate, why are you passing extra things? (e.g. method)